test/caesium/binding_test.clj

```diff
@@ -83,9 +83,12 @@
 (defn check-const-method
   "Check a method binding a const fn."
   [^Method method]
-  (let [rtype (.getGenericReturnType method)]
+  (let [rtype (.getGenericReturnType method)
+        int-funcs #{"sodium_init" "crypto_pwhash_argon2i_alg_argon2i13"
+                    "crypto_pwhash_alg_default" "crypto_pwhash_alg_argon2i13"
+                    "crypto_pwhash_argon2id_alg_argon2id13"}]
     (condp = rtype
-      Integer/TYPE (is (= "sodium_init" (.getName method)))
+      Integer/TYPE (is (true? (contains? int-funcs (.getName method))))
```

Comment:
This got eaten by the most recent changes: idiomatically, this should just be: `(is (int-funcs (.getName method)))` (and maybe `int-funcs` renamed to `int-func?` since you're effectively using it as a predicate.)

Reply:
that's a good point. I will take a look at it again and probably implement it as a int-func? as you suggest.